ddtrace/llmobs/_llmobs.py

```diff
@@ -327,6 +327,12 @@ def flush(cls) -> None:
         if cls.enabled is False:
             log.warning("flushing when LLMObs is disabled. No spans or evaluation metrics will be sent.")
             return
+
+        try:
+            cls._instance._evaluator_runner.periodic()
+        except Exception:
+            log.warning("Failed to run evaluator runner.", exc_info=True)
```

Comment:
(not actionable here) we should try to emit all the warning+ logs as apm telemetry events for us to monitor